config/routes.rb

```diff
@@ -3,4 +3,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+get "/guests/:id", to: "guests#show"
+post "/guests/:id", to: "guests#create"
```

Comment:
A more RESTful option here could be `post "/guests/:id/rooms", to: "guest_rooms#create"`
that way, we are reserving the guest specific paths and controller actions to actions that *only* deal with guests and updating its attributes, and the joins controller can handle this operation instead